backend/question-service/src/server.ts

```diff
@@ -0,0 +1,24 @@
+import express, { Application } from 'express';
+import mongoose from 'mongoose';
+import bodyParser from 'body-parser';
+import questionRoutes from './routes/questionRoutes';
+
+// Initialize the app
+const app: Application = express();
+const PORT = process.env.PORT || 3000;
+
+// Middleware
+app.use(bodyParser.json());
+
+// MongoDB connection
+mongoose.connect('mongodb://localhost:27017/peerprep', {})
```

Comment:
TODO: update the URI upon creation

Reply:
Sure , I will do it through env variable